src/language/class-modifiers.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Class modifiers
+description: Modifier keywords for class declarations to control external library access.
+---
+
+Class modifiers control how a class or a mixin can be implemented, extended,
+or mixed in. 
```

Comment:
maybe mention exhaustiveness?  "or mixed in, and whether they are considered for exhaustiveness checking in switches.".

Reply:
Hmm... Considering your comments, I'm actually leaning towards making this statement even more generic, something like: "Class modifiers control how a class or mixin can be _utilized_ by libraries other than the one they're defined in."

And then letting each modifier's section define what that "utilization" means to it. Or maybe that sounds terrible, or is that technically inaccurate? 

> maybe mention exhaustiveness?

This is a tricky one. My idea is that exhaustiveness checking is technically a product of the fact that `sealed` classes' ability to be utilized by outside classes is restricted, right? So, the intro statement is technically not excluding exhaustiveness checking, even though it is omitting it.

I wanted the purpose of this page to be kept really tight and just define the class modifiers, as agnostic from other topics as possible. Which is tough, considering the purpose of `sealed`. But in the end that decision was made specifically _because_ of `sealed`: because it would be easy for the definition of exhaustiveness checking to spread (and ultimately, be weakened because there'd be no single source of truth) across three different docs (this one, switch, patterns). 

[There will be dedicated content on exhaustiveness checking](https://github.com/dart-lang/site-www/issues/4635) added to the control flow section of the language tour that'll link to and from the `sealed` section here. I'll definitely be incorporating your reviews on emphasizing exhaustiveness checking in the `sealed` section on this page. But, I'm not sure I want to mention anything that specific at the top of this page, muddying what's technically the purpose statement of the page. I might change my mind as I go through the rest of these reviews though.